utils/build-script-impl

```diff
@@ -1884,7 +1884,10 @@ for deployment_target in "${HOST_TARGET}" "${CROSS_COMPILE_TOOLS_DEPLOYMENT_TARG
                     ( ! -z "${generator_output_path}" && ! -f "${generator_output_path}" ) ]] ; then
             set -x
             mkdir -p "${build_dir}"
-            (cd "${build_dir}" && "${CMAKE}" "${cmake_options[@]}" "${EXTRA_CMAKE_OPTIONS[@]}" "${source_dir}")
+            if [[ -n "${DISTCC}" ]]; then
+                EXTRA_DISTCC_OPTIONS="DISTCC_HOSTS=localhost,lzo,cpp"
+            fi
+            (cd "${build_dir}" && env $EXTRA_DISTCC_OPTIONS "${CMAKE}" "${cmake_options[@]}" "${EXTRA_CMAKE_OPTIONS[@]}" "${source_dir}")
```

Comment:
`"${EXTRA_DISTCC_OPTIONS}"` please.  I know it does not matter here, but we use it consistently everywhere so that we don't have to ask ourselves that question.